examples/current_sheet.py

```diff
@@ -1,9 +1,9 @@
 """
 ================================
-Plotting Field Lines from pfsspy
```

Comment:
Should this rewrite this example or should we create two seperate examples or a combined example?